packages/hardhat-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hardhat",
-  "version": "2.20.1",
+  "version": "2.21.0",
```

Comment:
is this correct?

Reply:
This is a temporary hack to allow for easy local testing via verdaccio. You can ignore everything in this commit: https://github.com/NomicFoundation/hardhat/pull/4655/commits/da8417f64a029b5cc13d556f910c070e8b2a3d37